src/NzbDrone.Api/Authentication/EnableAuthInNancy.cs

```diff
@@ -39,8 +39,8 @@ public void Register(IPipelines pipelines)
                 pipelines.EnableBasicAuthentication(new BasicAuthenticationConfiguration(_authenticationService, "Sonarr"));                
             }
 
-            pipelines.BeforeRequest.AddItemToEndOfPipeline(RequiresAuthentication);
-            pipelines.AfterRequest.AddItemToEndOfPipeline(RemoveLoginHooksForApiCalls);
+            pipelines.BeforeRequest.AddItemToEndOfPipeline((Func<NancyContext, Response>) RequiresAuthentication);
```

Comment:
I believe this is safe (and @ta264 already realized the other way was broken), running the PR build to confirm though.